pkg/annotation/annotation.go

```diff
@@ -6,14 +6,17 @@ const (
 	// Notes is for informational messages for resources generated by the operator.
 	Notes = "giantswarm.io/notes"
 
+	// AppOperatorPrefix the prefix for annotations that control logic inside app-operator.
+	AppOperatorPrefix = "app-operator.giantswarm.io"
+
 	// ChartOperatorPrefix is the prefix for annotations that control logic inside chart-operator.
 	ChartOperatorPrefix = "chart-operator.giantswarm.io"
 
 	// CordonReason is the name of the annotation that indicates
-	// the reason of why app-operator should not apply any update to this app CR.
-	CordonReason = "app-operator.giantswarm.io/cordon-reason"
```

Comment:
Better without prefix, since now we need to add chart-operator prefix sometimes.